### ISSUE TITLE
Illumina Genotype SNP arrays (Human)

### ISSUE THREAD

**Issue by raw937** [andrewparkermorgan/argyle#10]
Does your package help with this type of data?

**Comment by andrewparkermorgan** [comment]
Yes in principle `argyle` can work with human data (provided you have a file with the name, position and alleles for the markers on the array). How useful it will be depends on the format in which you receive the data, etc.

Commercial arrays for humans are mostly in the >500K marker range, so if you have more than a couple of hundred samples you probably should look for other tools that are optimised for larger genotype matrices.

**Comment by raw937** [comment]
Thank you! 
We have these idat formats (green/red). No idea how to look at quality, extract SNPs, and CNV from them. 
I just want a quick plot or something to tell me whether they are junk or not? 
Is this the best one? https://github.com/andrewparkermorgan/argyle/blob/master/vignettes/argyle.Rmd

**Comment by andrewparkermorgan** [comment]
This is not the package for you. The `*.idat` files are raw intensity values from the chip; getting from there to genotype calls and/or normalized intensities like you would use for CNV detection requires the [ Illumina array analysis platform](https://support.illumina.com/array/array_software/illumina-array-analysis-platform.html)
